Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -7,6 +7,16 @@
 // license that can be found in the LICENSE file.
 
 import UIKit
+import Dcrlibwallet
+
+enum Filter: Int{
```

Comment:
Space before the opening brace please.

Reply:
Looking at this again, it could conflict with the constants declared in Dcrlibwallet. At any rate, it is duplicating that information when those constants could be used directly.